labgrid/pytestplugin/reporter.py

```diff
@@ -21,6 +21,40 @@ def bold(text):
 def under(text):
     return "\033[4m{}\033[0m".format(text)
 
+@pytest.hookimpl()
+def pytest_collection_modifyitems(config, items):
```

Comment:
As this has nothing to do with the reporter, this should go into a separate file. Maybe `collection.py`?

Reply:
Maybe its a good idea to move the `pytest_configure` function as well into the __init__.py?